runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/BatchModeExecutionContext.java

```diff
@@ -543,6 +547,16 @@ public Long extractThrottleTime() {
         totalThrottleTime += httpClientApiThrottlingTime.getCumulative();
       }
 
+      CounterCell bigqueryStreamingInsertThrottleTime =
+          container.tryGetCounter(
+              MetricName.named(
+                  BIGQUERY_STREAMING_INSERT_THROTTLE_TIME_NAMESPACE,
+                  BIGQUERY_STREAMING_INSERT_THROTTLE_TIME_NAME));
```

Comment:
Can we use the same name as above ("cumulativeThrottlingSeconds") and move it to a constant (and also do ms to sec conversion when setting) ?

Reply:
+1
It would be great if we could use the same constant for all three use cases.

Reply:
Here we can use seconds, but on the streaming side msec is needed. That's the reason why I kept msec.

For consistency, we can change all counters to use msec originally, and do msec to sec conversion here. WDYT?

Reply:
Hi @chamikaramj @ihji ! I have made the change such that BQ, GCS, and Datastore all report throttled time in milliseconds at the beginning, and they now share the common counter name for consistency. The millisecond to second conversion is done only when later a throttled time in second is expected by worker side code. PTAL